### PR TITLE
(MAINT) Add Ruby 2.5 to Travis and Appveyor config

### PR DESCRIPTION
**Pull request by scotje**
Also moves "primary" Ruby version in Travis to 2.4.3

### CODE REVIEW

**Comment by scotje** [comment]
Ruby 2.5 cells will be red until #459 is merged.

File: appveyor.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 environment:
   matrix:
```
Comment (Iristyle) [inline appveyor.yml]: can we add `DEBUG_RESOLVER=1` to the environment?

That should show us a whole ton of output for the dependency resolution

**Comment by scotje** [comment]
Need to split the `path_or_url` stuff out into it's own PR and clean up a couple other things, but this is mostly working now!

**Comment by scotje** [comment]
Should go green after #462 is merged.

**Pull request merged**